src/componentes/card/card.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { AiFillStar} from 'react-icons/ai';
+import './card.css';
+import Location from "../location/Location";
+
+function Card({ rating: {name, city, country, rating} }) {
```

Comment:
El cambio del nombre de la prop se vería reflejado aqui:
```suggestion
function Card({ local: {name, city, country, rating} }) {
```